### PR TITLE
Updating text and screenshots

### PR DESCRIPTION
**Pull request by francisco-milan**
Updating text and screenshots as per the changes in #6759

### CODE REVIEW

File: src/user-guide/setup/settings-global.md [hunk 1]
```diff
@@ -4,14 +4,15 @@ category: setup
 order: 40
 ---
 
-Global settings in abapGit are valid system-wide and for all users. You can maintain the settings from the repository list or repository view by selecting "Settings > Global".
+Global settings in abapGit are valid system-wide and for all users. You can maintain the settings from the repository list or repository view by selecting "Global Settings" > "Global".
+
+![image](https://github.com/abapGit/docs.abapgit.org/assets/36721657/d9d2793d-cc38-4c55-adf0-792b879dddde)
```
Comment (larshp) [inline src/user-guide/setup/settings-global.md:9]: we want the screenshots in real git, ie. the /img/ folder

**Review by larshp — CHANGES_REQUESTED** [review]

File: src/user-guide/setup/settings-global.md (hunk 1 above)
Comment (larshp) [reply src/user-guide/setup/settings-global.md:9]: easiest is to start vscode, and do the editing from there

Comment (francisco-milan) [reply src/user-guide/setup/settings-global.md:9]: Hi @larshp,

    It looks like I don't have access to the /img/ folder
image

Comment (mbtools) [reply src/user-guide/setup/settings-global.md:9]: Images are stored in `https://github.com/abapGit/docs.abapgit.org/tree/main/src/.vuepress/public/img`

You can then refer to them, for example, like this on a doc page:

`abapGit`

Comment (francisco-milan) [reply src/user-guide/setup/settings-global.md:9]: Thanks @mbtools,

   I tried to upload a new file with a screenshot...

image

... but I got this message:

image

Comment (larshp) [reply src/user-guide/setup/settings-global.md:9]: choose the right branch?

Comment (mbtools) [reply src/user-guide/setup/settings-global.md:9]: you're trying to upload to the original repo but the changes need to be in your fork 😉

Comment (larshp) [inline src/user-guide/setup/settings-global.md:9]: nice, then replace these with the img

Comment (francisco-milan) [reply src/user-guide/setup/settings-global.md]: Got it! Thanks a lot for your help, now the screenshots and the links are updated.

Best,

Francisco Milán.

**Review by larshp — APPROVED** [review]

**Pull request merged**